lib/mongo/error/operation_failure.rb

```diff
@@ -52,6 +52,14 @@ class OperationFailure < Error
 
       def_delegators :@result, :operation_time
 
+      # @return [ Fixnum ] code The error code parsed from the document.
```

Comment:
Given that `Fixnum` is deprecated in favor of `Integer` in newer Ruby versions and that `#is_a?` will still return true for integers before this change, I think we should just document this as returning `Integer` (here and elsewhere in the change).

Reply:
OK.